src/appleseed/renderer/meta/tests/test_backwardlightsampler.cpp

```diff
@@ -42,7 +42,7 @@ using namespace renderer;
 
 TEST_SUITE(Renderer_Kernel_Lighting_BackwardLightSampler)
 {
-    TEST_CASE(HasLightsOrEmittingTriangles_GivenEmptyScene_ReturnsFalse)
+    TEST_CASE(HasLightsOrEmittingShapes_GivenEmptyScene_ReturnsFalse)
```

Comment:
The name of this test was wrong, it should be `HasLights_GivenEmptyScene_ReturnsFalse`.